gui/src/components/ProcessList.svelte

```diff
@@ -59,6 +60,14 @@
   --panel-padding="0 1rem"
 >
   <div slot="actions">
+    <IconButton
+      tooltip="Workspace details"
+      on:click={() => {
+        router.push(`#/workspaces/${encodeURIComponent(workspaceId)}/info`);
+      }}
+    >
+      <PreferencesSvg />
```

Comment:
_nit:_ I've set a convention of calling all the SVGs `SomeNameSVG.svelte` and using the fully capitalized `<SomeNameSVG />` for the component instances. Just keeps the names predictable (and identical between filename and component name)